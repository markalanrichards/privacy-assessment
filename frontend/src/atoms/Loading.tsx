import * as React from "react";

export interface AtomProps {}

export const Loading: React.SFC<AtomProps> = () => <p>Loading...</p>;
