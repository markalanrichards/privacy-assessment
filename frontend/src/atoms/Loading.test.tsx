import * as React from "react";
import { shallow, mount } from "enzyme";
import { Loading } from "./Loading";

test("error - loads no message", () => {
  const wrapper = shallow(<Loading />);
  expect(wrapper.text()).toBe("Loading...");
  expect(wrapper).toMatchSnapshot();
});
