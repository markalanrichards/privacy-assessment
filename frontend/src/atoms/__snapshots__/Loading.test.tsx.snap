// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error - loads no message 1`] = `
<p>
  Loading...
</p>
`;
